lib/Github/Api/Repo.php

```diff
@@ -62,13 +62,15 @@ public function all($id = null)
     /**
        * List all repositories owned by the user.
        *
-       * @link https://developer.github.com/v3/repos/#list-all-public-repositories
+       * @link https://developer.github.com/v3/repos/#list-your-repositories
        *
        * @param int|null $id The integer ID of the last Repository that you’ve seen.
+       * @param string|all $visibility The repo visibility
+       * @param int|null $id The integer ID of the last Repository that you’ve seen.
        *
        * @return array list of users found
        */
-      public function owned($id = null)
+      public function owned($visibility = all, $sort = full_name, $id = null)
```

Comment:
Lets to a `$param` like other endpoints: https://github.com/KnpLabs/php-github-api/blob/master/lib/Github/Api/Issue.php#L32